src/main/resources/templates/fragments/userBar.html

```diff
@@ -9,32 +9,25 @@
   <nav class="content">
     <ul class="mobile-hidden" id="navigation">
       <li>
-        <ul class="mobile-hidden"
-            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-          <li class="user" id="signed-in-user" th:text="${userModel.getId() + ':'}"></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your details - top menu" id="your-details"
-              th:href="@{{chsAccountUrl}/user/account(chsAccountUrl=${@environment.getProperty('account.url')})}">Your
-            details
-          </a></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your Applications" id="your-applications"
-              th:href="@{{manageApplicationsUrl}/(manageApplicationsUrl=${@environment.getProperty('manageApplications.url')})}">Your
-            Applications
-          </a></li>
+        <ul th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+            th:remove="tag">
+          <li class="user" id="signed-in-user" th:text="${userModel.getId()}"></li>
+          <li><a class="govuk-link"
+                 th:href="@{{forum}(forum=${@environment.getProperty('developerforum.url')})}">Developer
+            hub forum</a></li>
           <li><a class="piwik-event" data-event-id="Sign out" href="/signout" id="user-signout">Sign
             out
           </a></li>
         </ul>
       </li>
-      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-        <!--  -->
-        <a class="piwik-event govuk-link" data-event-id="Sign In" href="/signin" id="user-signin">Sign
-          in /
-          Register</a>
-      </li>
+      <ul th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+          th:remove="tag">
+        <li><a class="govuk-link"
```

Comment:
Sorry to be a pain Jon, but it should be Sign in/Register first then dev hub forum. Please check prototype.

Reply:
After speaking with michael gearon, he has agreed that we should use a layout that is consistent between logged in and logged out, that places the developer forum on the far right.